tobiasHe/assignments/slidingwindow/lc53/lc53.js

```diff
@@ -1,19 +1,46 @@
 function max_subarray_size_k(k, arr) {
-  // TODO: Write your code here
```

Comment:
Where is the love for high level overview/pseudo code?? #LanguageAgnostic #HelpMeHelpYou